src/libgrate/matrix.c

```diff
@@ -143,7 +143,7 @@ void mat4_perspective(struct mat4 *m, float fov, float aspect,
 	m->xx = cotangent / aspect;
 	m->yy = cotangent;
 	m->zz = -(far + near) / depth;
-	m->zw = -1.0f;
-	m->wz = -2.0f * near * far / depth;
+	m->zw = -2.0f * near * far / depth;
+	m->wz = -1.0f;
```

Comment:
We are using column-major notation for the mats, but perspective matrix was defined as row-major (transposed).

I'm not familiar with GLM, looks like its perspective matrix is same OpenGL-standard we had (row-major). What would be the benefits of switching to GLM? Seems it's C++ only.